lib/matplotlib/tests/test_axes.py

```diff
@@ -4282,9 +4282,16 @@ def _helper_y(ax):
 
 
 @cleanup
-def test_broken_barh_empty():
+def test_adjust_numtick_aspect():
```

Comment:
It looks like this removes the test_broken_barh_empty test because the rest accidentally got duplicated in an earlier merge